app/assets/javascripts/channels/reports.coffee

```diff
@@ -0,0 +1,44 @@
+jQuery(document).on "ready page:load", ->
```

Comment:
lý do gì sử dụng coffee ? để múa lửa thì áp dụng cho toàn bộ js trong project đi!

Reply:
Cái đó em tham khảo action cable ở trang https://edgeguides.rubyonrails.org/action_cable_overview.html
để làm comment chứ biết nhiều đâu mà ap dụng các file js còn lại chứ anh :(